src/sentry/snuba/outcomes.py

```diff
@@ -460,3 +464,33 @@ def massage_outcomes_result(
         del result["intervals"]
     del result["query"]
     return result
+
+
+def run_metrics_outcomes_query(
+    query: QueryDict, organization, projects, environments
+) -> dict[str, list]:
+    start, end = get_date_range_from_params(query)
+    interval = parse_stats_period(query.get("interval", "1h"))
+
+    # TODO(metrics): add support for reason tag
+    plan = MetricsQueriesPlan().apply_formula("sum(c:metric_stats/volume@none) by (outcome.id)")
+    rows = run_metrics_queries_plan(
+        plan,
+        start=start,
+        end=end,
+        interval=int(3600 if interval is None else interval.total_seconds()),
+        organization=organization,
+        projects=projects,
+        environments=environments,
+        referrer="outcomes.timeseries",
+    ).apply_transformer(MetricsStatsTransformer())
+
+    # Creating a dummy query definition to pass to the _format_rows function
+    copied = query.copy()
+    copied["category"] = "error"
```

Comment:
Could you elaborate why this is needed?

Reply:
`_format_rows` requires `QueryDict` as a param, and `QueryDict` contains a stats category. However `metrics` is not a valid stats category as of now. Possible solutions here would be to add metrics as a category or to refactor `_format_rows` not to require the entire `QueryDict`. We will do either one or the other in a follow up in which we will also add support for other params of this API.

I will add a code comment explaining this

Reply:
Great!